src/index.html

```diff
@@ -17,6 +17,10 @@
     />
   </head>
   <body>
-    <h1>Stop watch</h1>
+    <div class="stopwatch stopwatch--speed-up">
```

Comment:
BEM naming convention error: the modifier class 'stopwatch--speed-up' should be used with the block class 'stopwatch', so the correct class attribute should be 'stopwatch stopwatch--speed-up'.